catalog/env.template

```diff
@@ -117,3 +121,7 @@ SQLALCHEMY_SILENCE_UBER_WARNING=1
 
 AIRFLOW_VAR_AIRFLOW_RDS_ARN=unset
 AIRFLOW_VAR_AIRFLOW_RDS_SNAPSHOTS_TO_RETAIN=7
+
+# Whether to toggle production CloudWatch alarms when running a data refresh DAG.
+# Used to prevent requiring AWS credentials when running locally.
+AIRFLOW_VAR_TOGGLE_CLOUDWATCH_ALARMS=true
```

Comment:
For defaults for local development (e.g. contributors), this should definitely default to false:
```suggestion
AIRFLOW_VAR_TOGGLE_CLOUDWATCH_ALARMS=false
```

In fact, this could probably be removed from the `env.template` entirely if it is put into the DAG documentation instead (like [`SLACK_MESSAGE_OVERRIDE`](https://github.com/WordPress/openverse/blob/ee77ef5fe752fa5f4f89ca6d9c8f5af7fd4816ba/catalog/dags/common/slack.py#L15)).

Reply:
From other projects, I was used to the idea of making the defaults the values used in production, but it's true that the approach of the catalog is different. Change included!

Also, given it defaults to true in the function where it's used, I think it would be need it for local development, as you said.